Formula/duplicity.rb

```diff
@@ -324,10 +325,46 @@ def install
     # OSX doesn't provide a /usr/bin/python2. Upstream has been notified but
     # cannot fix the issue. See:
     #   https://github.com/Homebrew/homebrew/pull/34165#discussion_r22342214
-    inreplace "#{libexec}/bin/duplicity", "python2", "python"
+    inreplace libexec/"bin/duplicity", "python2", "python"
+
+    # We prefer gpg2, so set a default. This can still be overriden by
+    # passing the `--gpg-binary=` command line flag.
+    inreplace libexec/"lib/python2.7/site-packages/duplicity/globals.py",
+              "gpg_binary = None", "gpg_binary = 'gpg2'"
```

Comment:
I'm mostly appreciative that when Debian leads the way on things that carries more weight, and upstream developers take notice. We've had less "natural" (i.e. unprompted) success using Homebrew's muscle in the same way. Currently `gnupg` conflicts with MacGPG2 as well, because of the `gpg` symlink situation, whereas `gnupg2` doesn't. That's something that needs to be considered, given we're introducing more formal support for MacGPG2 here.